### PR TITLE
feat: fix to unoThemes issues

### PR DESCRIPTION
**Pull request by Arieldelossantos**
GitHub Issue: #

## Proposed Changes

 - [x] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## Description

Fix canary issues relating to unothemes namespace change (https://github.com/unoplatform/Uno.Themes/blob/master/doc%2Fmaterial-migration.md#updating-to-unothemes-v50 )

## Impact on version

- [ ] **Major**
  - The template structure was changed.
- [ ] **Minor**
  - New functionalities were added.
- [ ] **Patch**
  - A bug in behavior was fixed.
  - Documentation was changed.

### Always applicable
No matter your changes, these checks always apply.
- [ ] Your conventional commits are aligned with the **Impact on version** section.
- [ ] Updated [CHANGELOG.md](../CHANGELOG.md).
  - Use the latest Major.Minor.X header if you do a **Patch** change.
  - Create a new Major.Minor.X header if you do a **Minor** or **Major** change.
  - If you create a new header, it aligns with the **Impact on version** section and matches what is generated in the build pipeline.
- [ ] Documentation files were updated according with the changes.
  - Update `README.md` and `TemplateConfig.md` if you made changes to **templating**.
  - Update `AzurePipelines.md` and `APP_README.md` if you made changes to **pipelines**.
  - Update `Diagnostics.md` if you made changes to **diagnostic tools**.
  - Update `Architecture.md` and its diagrams if you made **architecture decisions** or if you introduced new **recipes**.
  - ...and so forth: Make sure you update the documentation files associated to the recipes you changed. Review the topics by looking at the content of the `doc/` folder.
- [ ] Images about the template are referenced from the [wiki](https://github.com/nventive/UnoApplicationTemplate/wiki/Images) and added as images in this git.
- [ ] TODO comments are hints for next steps for users of the template and not planned work.

### Contextual
Based on your changes these checks may not apply.
- [ ] Automated tests for the changes have been added/updated.
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj [hunk 1]
```diff
@@ -86,8 +86,9 @@
 		<PackageReference Include="ReviewService.NativePrompters" Version="1.0.0" />
 		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
 		<PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
-		<PackageReference Include="Uno.Material.WinUI" Version="4.0.6" />
-		<PackageReference Include="Uno.Toolkit.WinUI.Material" Version="5.0.17" />
+		<PackageReference Include="Uno.Material.WinUI" Version="4.2.0-dev.18" />
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj:89]: Can we use stable (non pre-release) instead?

**Comment by jeanplevesque** [comment]
what's wrong with the commits of this PR? Some are from 2021?

**Comment by Arieldelossantos** [comment]
created in the Wrong repo

File: src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj (hunk 1 above)
Comment (Arieldelossantos) [reply src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj:89]: i closed this one an reopened on the Fork one

**Pull request closed without merging**